pandas/io/tests/test_excel.py

```diff
@@ -557,6 +557,12 @@ def test_read_excel_multiindex(self):
         actual = read_excel(mi_file, 'mi_column_name', header=[0,1], index_col=0)
         tm.assert_frame_equal(actual, expected)
 
+        # Issue #11317
+        expected.columns = mi.set_levels(['1','2'],level=1).set_names(['c1', 'c2'])
```

Comment:
show the construction of expected here (I know it is probably above), but nice to have it actually here